zookeeper/Dockerfile

```diff
@@ -0,0 +1,39 @@
+FROM fedora:26
+
+RUN dnf -y install java-1.8.0-openjdk gettext hostname && dnf clean all -y
+
+# set Kafka home folder
+ENV ZOOKEEPER_HOME=/opt/zookeeper
+
+# Add zookeeper group / user
+RUN groupadd -r -g 1001 zookeeper && useradd -r -m -u 1001 -g zookeeper zookeeper
+
+# Set Zookeeper version
+ENV ZOOKEEPER_VERSION=3.4.10
+
+# Set MD5checksum
+ENV CHECKSUM="e4cf1b1593ca870bf1c7a75188f09678  zookeeper-3.4.10.tar.gz"
+
+# Set from build args
+ARG version=latest
+ENV VERSION ${version}
+
+WORKDIR $ZOOKEEPER_HOME
+
+# downloading/extracting Apache Zookeeper
+RUN curl -O http://www.eu.apache.org/dist/zookeeper/zookeeper-$ZOOKEEPER_VERSION/zookeeper-$ZOOKEEPER_VERSION.tar.gz  \
+    && echo $CHECKSUM > zookeeper-$ZOOKEEPER_VERSION.tar.gz.md5                                     \
```

Comment:
We should either just have a single space before the `\` or properly align the `\`. Personally I prefer the former.

Reply:
One space before \ seems to be already used in scripts. Change the Dockerfiles for Kafka and Zookeeper to match this style.